core/src/test/resources/sample_bank_account/bank.proto

```diff
@@ -220,6 +220,39 @@ message Transaction {
    required bool isValid = 9;
 }
 
+message Numerics {
+
+   /**
+    * @Field(store = Store.YES)
+    */
+   required int32 simpleByte = 1;
```

Comment:
I know this is just a test, but I would prefer if we used `optional` instead of `required` as recommended by the ProtoBuf 3 spec

Reply:
Updated.